readthedocs/api/v2/views/integrations.py

```diff
@@ -271,6 +289,21 @@ def handle_webhook(self):
                 raise ParseError('Parameter "ref" is required')
         if event in (GITHUB_CREATE, GITHUB_DELETE):
             return self.sync_versions(self.project)
+
+        if (
+            event == GITHUB_PULL_REQUEST and
+            self.data['action'] in [GITHUB_PULL_REQUEST_OPEN, GITHUB_PULL_REQUEST_SYNC]
+        ):
+            try:
+                identifier, verbose_name = self.get_external_version_data()
+                external_version = get_or_create_external_version(
+                    self.project, identifier, verbose_name
+                )
+                return self.get_response_push(self.project, external_version.verbose_name)
```

Comment:
To use the pre-written `self.get_response_push()` we can not pass a version instance so here I passed the verbose name but I think it will be much better if we pass the instance. to do this we need to create another function and not use `self.get_response_push()` we might use that function to send `triggered` status on the PR using the `Github Status API`. I would like to get you thought on this :)

Reply:
Yea, we can use whatever makes sense. If it's doing something different than existing code, don't feel bad coming up with a new method.